pkg/render/intrusion_detection.go

```diff
@@ -492,6 +503,17 @@ func (c *intrusionDetectionComponent) intrusionDetectionClusterRole() *rbacv1.Cl
 			},
 			Verbs: []string{"get"},
 		},
+		{
```

Comment:
is this always needed? Or just when running the webhook controller?

Reply:
This is required only when running the webhook controller. Changed